SU2_CFD/src/CMarkerProfileReaderFVM.cpp

```diff
@@ -95,24 +105,40 @@ void CMarkerProfileReaderFVM::ReadMarkerProfile() {
         }
         profileTags.push_back(text_line.c_str());
 
+        /*--- read NROW ---*/
         getline (profile_file, text_line);
         text_line.erase (0,5); numberOfRowsInProfile[iMarker] = atoi(text_line.c_str());
 
+        /*--- read NCOL ---*/
         getline (profile_file, text_line);
         text_line.erase (0,5); numberOfColumnsInProfile[iMarker] = atoi(text_line.c_str());
 
-        /*--- Skip the data. This is read in the next loop. ---*/
+        /*--- read the column format description. This line is not required, so if we cannot find it, we just continue  ---*/
+        getline (profile_file, text_line);
+        string::size_type dataheader = text_line.find ("# COORD",0);
+        if (dataheader == 0) {
+          skip = 0;
+        } else {
+          /*--- no header, but we have read a line, so we have to read one line less data ---*/
+          skip = 1;
+        }
 
-        for (unsigned long iRow = 0; iRow < numberOfRowsInProfile[iMarker]; iRow++) getline (profile_file, text_line);
+        /*--- Skip the data. This is read in the next loop. ---*/
+        
+        for (unsigned long iRow = 0; iRow < (numberOfRowsInProfile[iMarker]-skip); iRow++) getline (profile_file, text_line);
 
       }
     } else {
-      SU2_MPI::Error("While opening profile file, no \"NMARK=\" specification was found", CURRENT_FUNCTION);
+      //cout << "inlet profile reader is ignoring line: " << text_line << endl;
     }
   }
```

Comment:
in that case I would remove the whole `else ` statement ... unless there are further plans